app/helpers/title/title_helper.rb

```diff
@@ -8,7 +8,15 @@ def title(additional_context = {})
     class PageTitle
       def initialize(controller_path, action_name, context)
         @controller_path = controller_path
-        @action_name = action_name
+        @action_name = case action_name
+          when 'create'
+            'new'
+          when 'update'
+            'edit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.